labs/lab-01/tasks/len-xor/tests/test_len_xor.c

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+#include "graded_test.h"
+#include "../support/len_xor.h"
+
+#define SMALL_OUT	"small.out"
+#define SMALL_REF	"small.ref"
+#define MEDIUM_OUT	"medium.out"
+#define MEDIUM_REF	"medium.ref"
+#define BIG_OUT		"big.out"
+#define BIG_REF		"big.ref"
+
+int fd, ex_stdout;
+
+size_t read_buffer(int fd, void *buf, size_t size)
+{
+	size_t bytes_read, total_bytes_read = 0;
+
+	while (1) {
+		bytes_read = read(fd, buf + total_bytes_read, size - total_bytes_read);
+		if (bytes_read <= 0)
+			break;
+
+		total_bytes_read += bytes_read;
+		if (total_bytes_read == size)
+			break;
+	}
+
+	return total_bytes_read;
+}
+
+void redirect_stdout(char *file_name)
```

Comment:
```suggestion
static void redirect_stdout(char *file_name)
```
Make all unexported functions `static`. Think of them as "private" functions. Do this for all test files.